appsTransfer/src/main/java/org/kinecosystem/appstransfer/presenter/SenderServiceBinder.kt

```diff
@@ -0,0 +1,147 @@
+package org.kinecosystem.appstransfer.presenter
+
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+import android.content.ServiceConnection
+import android.content.pm.ResolveInfo
+import android.os.Handler
+import android.os.IBinder
+import android.os.Looper
+import android.util.Log
+import org.kinecosystem.common.base.Consts
+import org.kinecosystem.transfer.receiver.service.ServiceConfigurationException
+import org.kinecosystem.transfer.repositories.KinTransferCallback
+import org.kinecosystem.transfer.sender.service.SendKinServiceBase
+import java.util.concurrent.Executors
+
+class SenderServiceBinder(private val context: Context?) {
```

Comment:
isn't the code here almost identical to the one used for appsDiscovery ? Can we reuse instead of copy ?